### PR TITLE
Improve how we deal with Range of prices and it's decimals on Cost_utils

### PR DESCRIPTION
**Pull request by bordoni**
After this we Removed support for the weird negative prices.:smile:

_Ref:_ [C#38228](https://central.tri.be/issues/38228)

### CODE REVIEW

File: src/Tribe/Cost_Utils.php [hunk 1]
```diff
@@ -35,11 +35,9 @@ public function get_all_costs() {
 		$costs = $wpdb->get_col( "
 			SELECT DISTINCT meta_value
 			FROM   {$wpdb->postmeta}
-			WHERE  meta_key = '_EventCost'
-			       AND LENGTH( meta_value ) > 0
-		" );
+			WHERE  meta_key = '_EventCost' AND LENGTH( meta_value ) > 0;" );
```
Comment (borkweb) [inline src/Tribe/Cost_Utils.php]: Keep this on separate lines.

Comment (bordoni) [inline src/Tribe/Cost_Utils.php]: ok :win

**Comment by bordoni** [comment]
Fixed!

File: src/Tribe/Cost_Utils.php [hunk 2]
```diff
@@ -35,11 +35,11 @@ public function get_all_costs() {
 		$costs = $wpdb->get_col( "
 			SELECT DISTINCT meta_value
 			FROM   {$wpdb->postmeta}
-			WHERE  meta_key = '_EventCost'
-			       AND LENGTH( meta_value ) > 0
-		" );
+			WHERE
+				meta_key = '_EventCost' AND
+				LENGTH( meta_value ) > 0;" );
```
Comment (borkweb) [inline src/Tribe/Cost_Utils.php]: Revert this back to the original and I'd say this is good to go.

Comment (borkweb) [inline src/Tribe/Cost_Utils.php]: Unless you aren't seeing what I'm seeing:

screen shot 2015-11-23 at 5 23 53 pm

**Comment by borkweb** [comment]
If the GitHub diff is busted cheap and there aren't actually any random spaces between characters, then this is good to go. I think what I'm seeing is suspect.

Fix it if it needs fixing. I'm throwing a merge label on this.

**Pull request merged**